.github/workflows/component-bumper.yml

```diff
@@ -0,0 +1,32 @@
+name: Go
+
+on:
+  schedule:
+    - cron:  '0 5 * * *'
+
+jobs:
+
+  build:
+    name: CNAO Component Bump Job
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.13
+      id: go
```

Comment:
We don't need to install golang since it get installed as part of `make auto-bumper` that will call install-go.sh

Reply:
right!